### PR TITLE
Updates for cumulus only provisioning

### PR DESCRIPTION
**Pull request by cjh1**

### CODE REVIEW

File: ansible/site.yml [hunk 1]
```diff
@@ -65,6 +65,11 @@
       girder_path: /opt/hpccloud/girder
 
   post_tasks:
+    - name: Allow logins to be three chars
+      lineinfile: 'dest=/opt/hpccloud/girder/girder/conf/girder.local.cfg regexp="^login_regex:" line="login_regex: \"^[a-z][\da-z\-]{2}[\da-z\-]*$\""'
+      become_user: hpccloud
+      tags: girder
+
```
Comment (kotfic) [inline ansible/site.yml:76]: Maybe consider [ini_file](http://docs.ansible.com/ansible/ini_file_module.html) module?

File: ansible/site.yml [hunk 2]
```diff
@@ -194,6 +201,8 @@
           parameters:
             key: "pvwproxy.proxy_file_path"
             value: "/opt/hpccloud/proxy"
+      when: not cumulus
```
Comment (kotfic) [inline ansible/site.yml:208]: Can these conditioned ```when: not cumulus``` tasks be placed into a separate ```hpccloud.yml``` and conditionally included?

**Review by kotfic — COMMENTED** [review]
It is a little strange that deploying with CUMULUS=1 the girder login is still hpccloud, the vm, user is hpccloud,  everything is installed under /opt/hpccloud, etc..

File: ansible/site.yml (hunk 1 above)
Comment (cjh1) [reply ansible/site.yml:76]: I will take a look

File: ansible/site.yml (hunk 2 above)
Comment (cjh1) [reply ansible/site.yml:208]: The particular order is required, so not sure we ca extract them out into another file.

**Comment by cjh1** [comment]
@kotfic PTAL

**Review by kotfic — APPROVED** [review]
LGTM

**Pull request merged**